src/Views/Staff.jsx

```diff
@@ -0,0 +1,67 @@
+import data from '../data/staff.json';
+import blankPfp from '../assets/img/discord-blank2.png';
+
+export const Staff = () => {
+    return (
+       <main className="has-text-centered title">
+           <h1 className="is-size-1 pagetitle">Codecademy Community Server Staff</h1>
+           <p className="is-size-4 mb-3">Meet our server staff!</p>
+           <section className="container has-background-white mb-6 p-3">
+               <h2 className="is-size-2">Admins</h2>
+               <p>Admins are the heads of our staff and the people who manage how the group is run.</p>
+               <div className="staff-list">
+                   {
+                       data["superadmins"].map((user) => {
```

Comment:
Ideally these would be ported over to a `<StaffCard />` Component too with data as props but I felt this PR was already fairly long and would be best addressed in a follow up.